### PR TITLE
makes people get hurt when thrown at each other

### PR DESCRIPTION
**Pull request by PKPenguin321**
Does the same 10 damage and Weaken(1) as when they're thrown at a wall but affects the thrown person and the person hit by the thrown person, plays a punch sound, and throws a message. Very funny when combined with the wizard repulse spell

:cl: PKPenguin321
add: It now hurts to get thrown into another person.
/:cl:

### CODE REVIEW

**Comment by Xhuis** [comment]
10/10 branch name

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -16,6 +16,14 @@
 	if(hit_atom.density && isturf(hit_atom))
 		Weaken(1)
 		take_organ_damage(10)
+	if(iscarbon(hit_atom))
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/carbon_defense.dm]: you're aware this hits them twice?

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 2]
```diff
@@ -16,6 +16,14 @@
 	if(hit_atom.density && isturf(hit_atom))
 		Weaken(1)
 		take_organ_damage(10)
+	if(iscarbon(hit_atom))
+		var/mob/living/carbon/victim = hit_atom
+		victim.Weaken(1)
+		Weaken(1)
+		victim.take_organ_damage(10)
+		take_organ_damage(10)
+		visible_message("<span class='danger'> [src] crashes into [victim], knocking them both over!</span>", "<span class='userdanger'>You violently crash into [victim]!</span>")
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/carbon_defense.dm]: why the space?

Comment (PKPenguin321) [inline code/modules/mob/living/carbon/carbon_defense.dm]: mistake

**Comment by MrPerson** [comment]
Thank god nobody tried to overcomplicate this. Good addition.

**Comment by Jordie0608** [comment]
I can't wait till someone kills a xeno by throwing corpses at it.

**Comment by Kromgar** [comment]
Add a .01% chance of embed

**Comment by evsey9** [comment]
Make it have  a chance of embedding

**Comment by PKPenguin321** [comment]
That's not how it works you mongs

**Comment by Kromgar** [comment]
JUST DO IT! DONT  LET YOUR DREAMS BE DREAMS!

**Comment by MMMiracles** [comment]
Yeah just do it I'm sure it'll be extremely straight forward and not cause any issues.

**Comment by KorPhaeron** [comment]
Didn't we have this at some point and then remove it

**Comment by PKPenguin321** [comment]
Not that I know of

**Comment by optimumtact** [comment]
embedding a mob into something would not work with the current embedding system.

**Comment by Kromgar** [comment]
We totally did have this i remembner

**Pull request merged**